src/index.js

```diff
@@ -146,10 +149,38 @@ class MTProto {
         api_hash,
         dc,
         updates,
-        storage: new Storage(dc.id, { customLocalStorage }),
+        storage: new Storage(dc.id),
       });
     }
   }
+
+  importAuthKeys(authKeysAsJson) {
+    const authKeys = JSON.parse(authKeysAsJson)
+    this.authStorageKeys.forEach(key => {
+      if (authKeys[key]) this.storage.set(key, authKeys[key])
+    })
+  }
+
+  getAuthKeys() {
+    const authKeys = {};
+    this.authStorageKeys.forEach(key => {
+      if (this.storage.get(key)) authKeys[key] = this.storage.get(key)
+    })
+    return JSON.stringify(authKeys);
+  }
+}
+
+MTProto.dcList = function(test = false) {
+  return test ? TEST_DC_LIST : PRODUCTION_DC_LIST;
+}
+
+MTProto.authStorageKeys = function(test = false){
```

Comment:
I'm still confused by the dual meaning of keys in this code.

Reply:
renamed to `authStorageProps`